Gordon360/Models/CCT/dbo/FacStaff.cs

```diff
@@ -138,11 +138,14 @@ public partial class FacStaff
     [Unicode(false)]
     public string Type { get; set; }
 
+    [Column(TypeName = "datetime")]
+    public DateTime? FirstHireDt { get; set; }
```

Comment:
This hasn't been added to the view in Production yet. Also, make sure permissions are correct (e.g. one of the students can view the new column) in Production

Reply:
This has now been done and verified.